securedrop/sass/source.sass

```diff
@@ -297,8 +297,24 @@ header
       padding-top: 0
       min-height: unset
 
-    p
+    &> section:not(#locales-menu):not(#flashed)
+
+      width: 100%
+      padding-top: 10px
+      min-height: unset
+
+      &:first-child
+        @include ltr
+          border-right: 0
+        @include rtl
+          border-left: 0
+
+      &#return-visit
+        padding-left: 50px
+
+    p, hr
       width: auto
+      max-width: 500px
```

Comment:
There's no `hr` left in the HTML, so that can be left out.

Wondering about the `max-width`, is that just an aesthetic choice?

Reply:
Carry-over from the previous rules in the `source_index` module.